src/components/album/AlbumChatBox.js

```diff
@@ -47,28 +47,44 @@ const AlbumChatBox = ({ spotifyUrl }) => {
   };
 
   useEffect(() => {
+    const generateMessageWithKey = (data) => {
+      const isOwnMessage = loggedInUserId === data.user._id;
+      return {
+        ...data,
+        isOwnMessage,
+        messageKey: uuidv4(),
```

Comment:
I don't believe that we still need to create our own `messageKey` here in the frontend? Now we are returning the full `ChatMessage` model from the backend which already has a unique key that we can tell React to use when rendering a list of messages (`_id`) .

Reply:
<img width="508" alt="1" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/95896159/cda901d2-248a-4049-a225-33801f220f27">